src/lib/sundials_callbacks.pxi

```diff
@@ -63,7 +69,7 @@ cdef inline void arr2nv_inplace(x, N_Vector out) noexcept:
     cdef void* data_ptr=PyArray_DATA(ndx)
     memcpy((<N_VectorContent_Serial>out.content).data, data_ptr, n*sizeof(realtype))
     
-cdef inline np.ndarray nv2arr(N_Vector v) noexcept:
```

Comment:
Isn't `noexcept` necessary for Cython 3.x?

Reply:
not for functions returning pure python objects

Reply:
Ahh, I see, thank you!